ui/app/AppLayouts/Wallet/panels/DerivedAddressesPanel.qml

```diff
@@ -15,7 +15,7 @@ import "../stores"
 Item {
     id: derivedAddresses
 
-    property int selectedAccountType: RootStore.defaultSelectedType
+    property int selectedAccountType: Constants.AddAccountType.GenerateNew
```

Comment:
are we sure about this? is it confirmed with wallet team?

Reply:
I haven't confirmed with wallet team because there is no functional change. It has the same value as before:
`readonly property int defaultSelectedType: SelectGeneratedAccount.AddAccountType.GenerateNew`